doc/examples/Repository.rst

```diff
@@ -49,3 +49,11 @@ Get all the labels of the repository
     Label(name="WIP")
     Label(name="bug")
     Label(name="documentation")
+
+Mark the notifications of the repository as read
+------------------------------------------------
+
+.. code-block:: python
+
+    >>> repo = g.get_repo("PyGithub/PyGithub")
+    >>> repo.mark_notification_has_read()
```

Comment:
I think there's a typo here, the new method's name is `repo.mark_notification_as_read `, however it says `..has_read` here

Reply:
good find!